core/optic/shared/src/main/scala/com/useoptic/types/capture/Capture.scala

```diff
@@ -12,9 +12,9 @@ case class GroupingIdentifiers(agentGroupId: String,
 case class ShapeHashBytes(bytes: Vector[Byte])
 
 @JSExportAll
-case class ArbitraryData(asShapeHashBytes: Option[ShapeHashBytes],
-                         asJsonString: Option[String],
-                         asText: Option[String])
+case class ArbitraryData(asShapeHashBytes: Option[ShapeHashBytes] = None,
```

Comment:
shouldn't we do companion object .empty here instead?

Reply:
It's more for the case where you want to create ArbitraryData and only want to specify one of the properties ie `ArbitraryData(asJsonString = Some(example))`

Reply:
Same principle - it would be better to have builders. ArbitraryData.fromJson(...)